armi/utils/tests/test_utils.py

```diff
@@ -232,22 +232,21 @@ def test_prependToList(self):
         a = ["Foo", "Bar"]
         b = [1, 2, 3]
         utils.prependToList(a, b)
-        self.assertTrue(a == [1, 2, 3, "Foo", "Bar"])
+        self.assertListEqual(a, [1, 2, 3, "Foo", "Bar"])
```

Comment:
Ah that answers my previous question.

Reply:
I tried to clean up the commit history after the fact but it looks like I missed this.